Tools/CMake/AMReXParallelBackends.cmake

```diff
@@ -217,6 +237,20 @@ if (AMReX_HIP)
    # missing gpu devices)
    target_compile_options(amrex
       PUBLIC
-      $<$<COMPILE_LANGUAGE:CXX>:-m64 --amdgpu-target=${AMReX_AMD_ARCH}> )
+      $<$<COMPILE_LANGUAGE:CXX>:-m64> )
+
+  if(${_this_comp} STREQUAL hipcc)
+      target_compile_options(amrex PUBLIC
+          $<$<COMPILE_LANGUAGE:CXX>:-m64 --amdgpu-target=${AMReX_AMD_ARCH}> )
```

Comment:
Note: we need to double-check how we can control the HIP config logic with our `AMReX_AMD_ARCH` settings, if provided by the user